### DISCUSSION TITLE
The icon in the top left corner is suggested to be Positron's instead of Code OSS

### DISCUSSION THREAD

**Discussion by chuxinyuan** [posit-dev/positron#3979] [Feedback]
Such as the question.

**Comment by juliasilge** [comment]
Can you share what OS you use, and perhaps a screenshot showing where you are seeing this pop up?

**Reply by chuxinyuan** [reply]
![202407120135](https://github.com/posit-dev/positron/assets/26940042/987cad50-76c2-4eec-9d2b-24751700fb02)

**Reply by juliasilge** [reply]
Oh man, we thought we had fixed this. I have reopened this issue so we can re-examine: https://github.com/posit-dev/positron/issues/2089

**Reply by petetronic** [reply]
This is a slightly different issue, so I've closed 2089 and opened a new issue #3996 to capture this scenario.
